src/main/java/Problem3/MovieComedy.java

```diff
@@ -21,5 +24,14 @@ public int getLateFeeInDollar() {
     @Override
     public int calcLateFees(int numOfDaysPastDue) {
         // homework
+        if(numOfDaysPastDue <= 0)
+        {
+            return 0;
+        }
+        else if(numOfDaysPastDue > 0)
```

Comment:
if code reaches this point, it will be numOfDaysPastDue > 0 for sure. so this "else if" is not really needed.